akd/src/tree_node.rs

```diff
@@ -258,16 +266,24 @@ impl TreeNode {
             e if e > 0 => e - 1,
```

Comment:
Why do we use only `e-1` here? Shouldn't this be the previous time this node was modified? A node could remain untouched for a period of time.

Reply:
I was confused by this previously too (maybe `target_epoch` can have a better name), but this value is simply used to compare against the last epoch stored by a node. If `target_epoch` is smaller, the older set of node attributes are used.


https://github.com/novifinancial/akd/blob/97b2e90589083fef4600f732618715fac29a24f2/akd/src/tree_node.rs#L122-L144